src/kubernetes.h

```diff
@@ -151,6 +186,16 @@ class KubernetesReader {
       version_to_kind_to_name_;
   // A memoized map from an encoded owner reference to the owner object.
   mutable std::map<std::string, json::value> owners_;
+  // Mutex for the service related caches.
```

Comment:
Let's add a blank line before this group of variables.

Reply:
done